app/renderer/components/ServerMonitor/ServerMonitor.js

```diff
@@ -71,9 +71,27 @@ class GetRawLogsButton extends Component {
   }
 }
 
+class ClearLogsButton extends Component {
+  static propTypes = {
+    serverStatus: PropTypes.string.isRequired,
+  }
+
+  render () {
+    const {serverStatus, clearLogs} = this.props;
+    if (serverStatus !== STATUS_STOPPED) {
+      return <Button className={styles.serverButton}
+              icon="pause-circle-o" 
+              onClick={clearLogs}>CLean Logs</Button>;
+    } else {
+      return null;
+    }
```

Comment:
I don't think the `else { return null; }` is necessary

(EDIT: I stand corrected. The render method breaks if it returns undefined)